scripts/globals/summon.lua

```diff
@@ -18,9 +18,28 @@ function AvatarPhysicalMove(avatar, target, skill, numberofhits, accmod, dmgmod,
     local dmg = avatar:getWeaponDmg()
     local minFstr, maxFstr = avatarFSTR(avatar:getStat(tpz.mod.STR), target:getStat(tpz.mod.VIT))
     local ratio = avatar:getStat(tpz.mod.ATT) / target:getStat(tpz.mod.DEF)
+    local lvluser = avatar:getMainLvl()
+    local lvltarget = target:getMainLvl()
+    local firstacc = acc + 100
+
+-- formula should be 75 + floor( (Accuracy - Evasion)÷2 ) - 2×(dLVL)
```

Comment:
Keep these inline with the rest of the section

```lua
    --- formula...
    local blabla
```

Reply:
attempted to resolve